snowflake/objects/database/recover/schema/parquet/table/enrolledparticipants_parquet.sql

```diff
@@ -0,0 +1,92 @@
+CREATE OR ALTER TABLE ENROLLEDPARTICIPANTS (
+    "id" NUMBER(38,0),
+    "CustomFields_Treatments_id" VARCHAR(16777216),
```

Comment:
Is there a reason we set `VARCHAR` for every str var to be the maximum possible length in Snowflake?

Reply:
Table data types are defined to be as large as possible without consuming extraneous storage or requiring additional processing. For more information, see [String & binary data types](https://docs.snowflake.com/en/sql-reference/data-types-text) and [Numeric data types](https://docs.snowflake.com/en/sql-reference/data-types-numeric).